### ISSUE TITLE
Finetuning a nework with batchnormalization

### ISSUE THREAD

**Issue by alevillard** [vlfeat/matconvnet#537]
Hi, I have trained a small googlenet with a large database, inserting batchnorm layers and it works. But now that I want to finetune it with a smaller database I have this error: 

ans = 

             name: 'bn0'
           inputs: {'input'}
          outputs: {'bn0'}
           params: {'bn0f'  'bn0b'  'bn0m'}
     inputIndexes: 1
    outputIndexes: 2
     paramIndexes: [1 2 3]
      forwardTime: []
     backwardTime: []
            block: [1x1 dagnn.BatchNorm]

ans =

   224   224     3    20

ans = 

             name: 'conv1'
           inputs: {'bn0'}
          outputs: {'conv1'}
           params: {'conv1f'  'conv1b'}
     inputIndexes: 2
    outputIndexes: 3
     paramIndexes: [4 5]
      forwardTime: []
     backwardTime: []
            block: [1x1 dagnn.Conv]

ans =

   224   224     3    20

Error using vl_nnconv
The number of filter groups does not divide the number of filters.

the starting architecture is this:

net = dagnn.DagNN() ;
	net.addLayer('bn0', dagnn.BatchNorm('numChannels', 3), {'input'}, {'bn0'}, {'bn0f', 'bn0b', 'bn0m'});

    net.addLayer('conv1', dagnn.Conv('size', [7 7 3 64], 'hasBias', true, 'stride', [2, 2], 'pad', [3 3 3 3]), {'bn0'}, {'conv1'},  {'conv1f'  'conv1b'});
	net.addLayer('relu1', dagnn.ReLU(), {'conv1'}, {'conv1x'}, {});
        net.addLayer('bn1', dagnn.BatchNorm('numChannels', 64), {'conv1x'}, {'bn1'}, {'bn1f', 'bn1b', 'bn1m'});

**Comment by alevillard** [comment]
To be sure that is not a problem of my database, I have tried to start the finetuning with the same database I had made the training, but I have still the same error, so it's a problem of my code, or something else, do you know what can it be?  I have done many times the finetuning before, but not with batch layers
